Sources/SWBTaskExecution/TaskActions/ClangCompileTaskAction.swift

```diff
@@ -610,9 +616,65 @@ public final class ClangNonModularCompileTaskAction: TaskAction {
     }
 }
 
+fileprivate func parseTraceData(_ data: Data) throws -> TraceData? {
+    if let jsonObject = try JSONSerialization.jsonObject(with: data) as? [String: Any],
```

Comment:
We don't use JSONSerialization directly in Swift Build, as it is Any-typed. When you need to decode _arbitrary_ JSON structures, use:

```swift
if let jsonObject = try PropertyList.fromJSONData(data)
```

...which will give you a strongly-typed JSON object tree. You can then access the version like:

```swift
jsonObject.dictValue?["version"]?.stringValue
```